bitnami/kafka/templates/statefulset.yaml

```diff
@@ -91,23 +91,20 @@ spec:
         - name: volume-permissions
           image: {{ include "kafka.volumePermissions.image" . }}
           imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
-          command:
```

Comment:
Why do you remove this block ?

Reply:
not sure, I think that was accidental when copying the changes I made to our clone of the chart. I'll restore them.